### DISCUSSION TITLE
Problem in committing the changes

### DISCUSSION THREAD

**Discussion by saloni0419** [ohcnetwork/care_fe#7299] [General]
Can anyone please help me on this I have solved the issue, i am not able to push it on Git Hub i am running the command right but on git commit it is showing no filed staged. I have forked, cloned right added a new branch, and git add. , git status. I have done all these still showing the error. Can you please help me
![image](https://github.com/coronasafe/care_fe/assets/100074110/3e1591ca-65cd-4044-977a-7857229b7232)

**Comment by aeswibon** [comment]
@saloni0419 The error is saying that you are trying to commit when there are no staged files. Can you check whether the changes are there when you are trying to commit?

**Comment by saloni0419** [comment]
@aeswibon yes am committing it when files are changed

**Comment by aeswibon** [comment]
@saloni0419 Let's hurdle on Slack. That way, I would be able to clearly understand the problem and would be able to help you debug it.

**Comment by saloni0419** [comment]
@aeswibon can you please give me link to join on slack

**Reply by aeswibon** [reply]
Slack link: https://slack.ohc.network/
